seastar/src/submit_to.hh

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "cxx-async/include/rust/cxx_async_seastar.h"
+#include "rust/cxx.h"
+#include "cxx_async_futures.hh"
+
+namespace seastar_ffi {
+
+namespace submit_to {
+
+typedef VoidFuture (*fn_once_caller_t)(const uint8_t*);
```

Comment:
Nit: please use `using` instead of `typedef`, it has much more sensible syntax:

```c++
using fn_once_caller_t = VoidFuture (*)(const uint8_t*);
```